examples/basic-udp.yaml

```diff
@@ -15,7 +15,7 @@ metadata:
   name: my-gateway
   namespace: default
 spec:
-  class: acme-lb
+  gatewayClassName: acme-lb
```

Comment:
Out of scope for this PR, but PTAL at https://github.com/kubernetes-sigs/service-apis/issues/356 when you have a moment.